static/app/plugins/sessionstack/components/settings.tsx

```diff
@@ -7,12 +7,12 @@ import DefaultSettings from 'sentry/plugins/components/settings';
 type Props = DefaultSettings['props'];
 
 type State = DefaultSettings['state'] & {
-  showOnPremisesConfiguration?: boolean;
+  showSelfHostedConfiguration?: boolean;
```

Comment:
This ancient plugin has a few config values that are only relevant on self-hosted. This is the source of the [screenshots that changed](https://storage.googleapis.com/sentry-visual-snapshots/getsentry/sentry/ba0d0b4168d6549d6b59de82677b9d247ca0540d/index.html).